tapeagents/prompting.py

```diff
@@ -61,7 +61,8 @@ def prompt_with_guidance(tape: Tape, guidance: str) -> Prompt:
 def view_to_messages(view: TapeView, agent: Agent | None = None) -> list[dict]:
     messages = []
     for step in view.steps:
-        assert not isinstance(step, (Pass, SetNextNode))
+        if isinstance(step, (Pass, SetNextNode)):
```

Comment:
before `Pass` and `SetNextNode` would not make it into the view, I suppose you changed that

Reply:
We've discussed this during the call, leaving it as it is.